src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -311,6 +312,7 @@ private function addRequestSection(ArrayNodeDefinition $rootNode)
 
     private function addTemplatingSection(ArrayNodeDefinition $rootNode)
     {
+        /** @deprecated since 2.7, will be removed in 3.0 */
```

Comment:
the ``->info()`` of the node should be updated to mark it as deprecated when dumping the config reference